index.html

```diff
@@ -6,6 +6,86 @@
   <title>Homework 4</title>
 </head>
 <body>
-  
+  <header> 
+    <h1>HTML5 Семантичні елементи</h1>
+     <p>
+      Семантичний елемент чітко описує його значення як для браузера, так i для розробника.
+      Приклади не <strong>семантичних</strong> елементів: &#x3C;div&#x3E;  i &#x3C;span&#x3E;- нічого не говорить про його вміст.
+      Приклади <strong>семантичних</strong> елементів: &#x3C;form&#x3E;, &#x3C;table&#x3E; i &#x3C;article&#x3E;- чітко визначає його зміст.
+     </p>
+
+     <h2>Підтримка браузерів</h2>
+      <img src="images/1520047697221.jpg" alt="various browers" width="200px">
+      <p>
+        Семантичні елементи HTML5 підтримуються y всіх сучасних браузерах.Крім того, ви можете "навчити"
+        старих браузерів, як обробляти "невідомі елементи".
+        Прочитайте про це в <a href="">підтримці браузера HTML5</a>.
+      </p>
+
+  </header>
+
+  <main>
+    <section>
+      <h2>Нові семантичні елементи в HTML5</h2>
+      <p>
+        HTML5 пропонує нові семантичні елементи для визначення різних частин
+        веб-сторінки:
+        <ul>
+        <li>&#x3C;section&#x3E;</li>
+        <li>&#x3C;article&#x3E;</li>
+        <li>&#x3C;header&#x3E;</li>
+        <li>&#x3C;footer&#x3E;</li>
```

Comment:
це мають бути якоря
https://www.geeksforgeeks.org/how-to-create-links-to-sections-within-the-same-page-in-html/

```suggestion
        <li><a href="#section">&#x3C;section&#x3E;</a></li>
        <li><a href="#article">&#x3C;article&#x3E;</a></li>
        <li><a href="#header">&#x3C;header&#x3E;</a></li>
        <li><a href="#footer">&#x3C;footer&#x3E;</a></li>
```